packages/terminal/src/browser/terminal-frontend-contribution.ts

```diff
@@ -50,6 +50,18 @@ export class TerminalFrontendContribution implements TerminalService, CommandCon
         @inject(WorkspaceService) protected readonly workspaceService: WorkspaceService
     ) { }
 
+    registerTermCommands(termWidget: TerminalWidget, commands: CommandRegistry) {
+        const CLEAR_OUTPUT: Command = {
+            id: 'terminal:' + termWidget.id + ':clearOutput',
```

Comment:
We should track the current (last focused) terminal similar to how we track the current editor and have a single command which is executed on a current terminal.

Reply:
Add tracking for current active terminal